src/test/java/ai/vespa/cloud/docsearch/VespaDocStagingSetup.java

```diff
@@ -0,0 +1,44 @@
+package ai.vespa.cloud.docsearch;
+
+import ai.vespa.feed.client.DocumentId;
+import ai.vespa.hosted.cd.Endpoint;
+import ai.vespa.hosted.cd.StagingSetup;
+import ai.vespa.hosted.cd.TestRuntime;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestReporter;
+
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@StagingSetup
+public class VespaDocStagingSetup {
+
+    final Endpoint endpoint = TestRuntime.get().deploymentToTest().endpoint("default");
+
+    @Test
+    @DisplayName("Feed and verify documents")
+    public void feedAndVerify(TestReporter report) throws Exception {
+        VespaDocTester tester = new VespaDocTester(endpoint, report);
+        tester.removeAllTestDocs();
+        tester.feedTestDocs();
+
+        Set<DocumentId> ids = tester.getTestDocIDs();
+        for (DocumentId id : ids) report.publishEntry(id.toString());
+        assertEquals(5, ids.size(), "test-documents.json should have 5 documents");
+
+        String allQuery = "select * from sources * where sddocname contains \"doc\";";
```

Comment:
; not needed anymore
```suggestion
        String allQuery = "select * from sources * where sddocname contains \"doc\"";
```